source/configuration_manual/event_export.rst

```diff
@@ -120,6 +120,10 @@ Currently, there are three transports:
 * `http-post` - send the serialized event as a HTTP POST payload to the URL
   specified in the ``transport_arg`` setting with a timeout specified by
   ``transport_timeout``
+* `file` - send serialized events to file or unix socket specified in
+  the ``transport_arg`` setting. For unix socket, the filename needs to be
+  prefixed with ``unix:``. ``transport_timeout`` is used to specify how
+  long the unix socket connection can take. Default is 250 milliseconds.
```

Comment:
I don't think this is true anymore? There should be separate "file" and "unix" transports.